file_catalog/server.py

```diff
@@ -139,6 +135,18 @@ def __init__(self, config, port=8888, db_host='localhost', db_port=27017, debug=
         )
         app.listen(port)
 
+    def print_config(self, config, port, debug, db_host, db_port, db_user, db_pass):
+        logger.info('db host: %s', db_host)
+        logger.info('db port: %s', db_port)
+        logger.info('db user: %s', db_user)
+        db_pass_hash = (hashlib.sha256(db_pass.encode('utf-8')).hexdigest() if db_pass else None)
+        logger.info('db pass hash: %s', db_pass_hash)
```

Comment:
Again, I probably wouldn't even log the hash here.

Reply:
The idea is not to print/log clear-text password, but still make it possible to verify that the correct password is used.